examples/logical_interconnect.rb

```diff
@@ -99,18 +112,18 @@
 oneview_logical_interconnect 'Encl1-LogicalInterconnectGroup1' do
   client my_client
   data(
-    'snmpConfiguration' => {
-      'snmpAccess' => ['172.18.6.15/24']
+    snmpConfiguration: {
+      snmpAccess: ['172.18.6.15/24']
     }
   )
   trap_destinations(
-    '172.18.6.16' => {
-      'trapFormat' => 'SNMPv1',
-      'communityString' => 'public',
-      'severities' => ['Critical', 'Major', 'Unknown'],
-      'vcmTraps' => ['Legacy'],
-      'enetTraps' => ['PortStatus', 'PortThresholds'],
-      'fcTraps' => ['Other']
+    '172.18.6.16': {
```

Comment:
I don't thik it will work

Reply:
I also had this doubt, but I tested it and Ruby converts that string into a symbol and that worked fine. But if this is strange, I can modify it.

Reply:
Chnage to string hash syntax `'x' => y`